runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1332,6 +1332,9 @@ void BaseLoader<LoaderDomain>::loadOperation(const Operator *op, ir::Graph &subg
     case BuiltinOperator::BuiltinOperator_CONCATENATION:
       loadConcatenation(op, subg);
       return;
+    case BuiltinOperator::BuiltinOperator_FLOOR:
+      loadOperationTo<ir::operation::Floor>(op, subg);
```

Comment:
@hasw7569 Can't we use `ElementwiseUnary`? (internal IR, cpu backend layer)

Reply:
Thanks for your opinion.
I will fix it soon.